web/src/app/browser/src/context/focusAssistant.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * The return object documented for
+ * https://help.keyman.com/DEVELOPER/ENGINE/WEB/16.0/reference/core/getUIState.
```

Comment:
```suggestion
 * https://help.keyman.com/developer/engine/web/16.0/reference/core/getUIState.
```

The UPPER/CASE is a bug in IIS, we don't want to propagate that.